packages/@react-aria/calendar/src/useCalendarBase.ts

```diff
@@ -82,19 +49,11 @@ export function useCalendarBase(props: CalendarPropsBase & DOMProps, state: Cale
   // Announce when the selected value changes
   useUpdateEffect(() => {
     if (selectedDateDescription) {
-      announce(selectedDateDescription, 'polite', 4000);
+      announce(selectedDateDescription);
```

Comment:
The change to use "polite" seemed to cause VoiceOver to double announce the selected date. Do you remember why you changed this?

Reply:
As I recall, the change to use polite with a delay was to ensure that a Selected Date or Selected Range gets announced, particularly when the focused date is not it the range. But it seems to be working ok without it.

Reply:
hmm ok. perhaps it's a bug in the announcer itself, I didn't look to closely since this appears to work ok.

Reply:
I'm not sure that it was an announcer problem, but perhaps more in the context of the DatePicker or DateRangePicker controls, where the Calendar gets auto-focused when the dialog opens, but if a range is selected, the user is not notified. I also want to confirm the behavior with NVDA and JAWS, which might announce the state change of currently focused cell, without announcing the range, but my local VM seems be corrupted.